### ISSUE TITLE
SubX: Tip in `subx help opcodes` doesn't work

### ISSUE THREAD

**Issue by tekknolagi** [akkartik/mu#11]
```
willow% ./subx help opcodes     
Opcodes currently supported by SubX:
  01: add r32 to rm32
  03: add rm32 to r32
... SNIP ...
  0f 8e: jump disp16 bytes away if lesser or equal (ZF is set or SF != OF)
  0f 8f: jump disp16 bytes away if greater (ZF is unset, SF == OF)
Run `subx help instructions` for details on words like 'r32' and 'disp8'.
willow% ./subx help instructions
No help found for 'instructions'
Available top-level topics:
  usage
  registers
  opcodes
  syntax
Please check your command for typos.
willow%
```

**Comment by tekknolagi** [comment]
Would it be worth introducing a global subcommand map of sorts?

**Comment by akkartik** [comment]
Fixed by 431f739ddca73eb105bced65c169963a335e3134.

Yup, we can consider a map. I don't know how many more we're going to have immediately, so seems lower priority than labels and so on. You know, features :)
